src/main/java/core/basesyntax/dao/fileWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.dao;
+
+public interface fileWriter {
```

Comment:
The interface name `fileWriter` does not follow Java naming conventions. Interface names should be in PascalCase, such as `FileWriter`.